website/docs/d/cognitive_account.html.markdown

```diff
@@ -43,6 +43,11 @@ The following attributes are exported:
 * `endpoint` - The endpoint of the Cognitive Services Account
 
 * `qna_runtime_endpoint` - If `kind` is `QnAMaker` the link to the QNA runtime.
+
+* `qna_search_endpoint_id` - If `kind` is `TextAnalytics` the ID to the Search service.
+
+-> **NOTE:** `qna_search_endpoint_id` is used for [the new version of QnA Maker](https://docs.microsoft.com/azure/cognitive-services/qnamaker/custom-question-answering), while `qna_runtime_endpoint` is used for [the old version of QnA Maker](https://docs.microsoft.com/azure/cognitive-services/qnamaker/overview/overview)
```

Comment:
This might benefit from a bit more information
```suggestion
-> **NOTE:** `custom_question_answering_search_service_id` is used for [Custom Question Answering, the renamed version of QnA Maker](https://docs.microsoft.com/azure/cognitive-services/qnamaker/custom-question-answering), while `qna_runtime_endpoint` is used for [the old version of QnA Maker](https://docs.microsoft.com/azure/cognitive-services/qnamaker/overview/overview)
```